cli/backend_base.py

```diff
@@ -645,6 +645,7 @@ def _check_config(self, keyfile):
         self._check_pnda_mirror()
         self.check_target_specific_config()
         self._check_data_volume_count()
+        self._ensure_certs()
```

Comment:
Can you just rename the function to have consistency with the function names above and something more relevant now. Something like: _check_security_material()